app/src/frontend/building/data-container.tsx

```diff
@@ -14,39 +41,55 @@ import InfoBox from '../components/info-box';
  *
  * @param WrappedComponent
  */
-const withCopyEdit = (WrappedComponent) => {
-    return class extends React.Component<any, any> { // TODO: add proper types
+const withCopyEdit = (WrappedComponent: React.ComponentType<CategoryViewProps>) => {
+    return class DataContainer extends React.Component<DataContainerProps, DataContainerState> {
```

Comment:
Make the local class named, so that the `DataContainer` class name displays e.g. in React Developer Tools in the browser (previously the node in the component tree displayed as an anonymous name with a name like `_a`)